lib/bitvec.cpp

```diff
@@ -137,3 +137,12 @@ unsigned bitvec::ffz(unsigned start) const {
 #endif
     return rv;
 }
+
+bool bitvec::is_contiguous() const {
+    // Empty bitvec is not contiguous
+    if (popcount() == 0)
+        return false;
+    if (max().index() - min().index() + 1 == popcount())
+        return true;
+    return false;
```

Comment:
Even better as then you don't need the check for `popcount() == 0` as when that is true, `max.index() == min.index() == -1;`.   Or is that too obscure?